packages/iTwinUI-react/src/core/Table/Table.tsx

```diff
@@ -708,25 +708,29 @@ export const Table = <
                       }}
                     >
                       {column.render('Header')}
-                      {(data.length !== 0 || areFiltersSet) && (
-                        <FilterToggle
-                          column={column}
-                          ownerDocument={ownerDocument}
-                        />
-                      )}
-                      {data.length !== 0 && column.canSort && (
-                        <div className='iui-cell-end-icon'>
-                          {column.isSorted && column.isSortedDesc ? (
-                            <SvgSortDown
-                              className='iui-icon iui-sort'
-                              aria-hidden
-                            />
-                          ) : (
-                            <SvgSortUp
-                              className='iui-icon iui-sort'
-                              aria-hidden
+                      {(column.Filter || column.canSort) && (
+                        <div className='iui-table-header-actions-container'>
```

Comment:
Doesn't look nice 
![image](https://user-images.githubusercontent.com/36186912/169040829-5899b873-1fab-49f4-a403-c4ad1da6df0a.png)

Reply:
But it will still look bad with regular non-resizable columns

Reply:
I added inline style to undo flex-wrap for now. We can merge this update and fix table with future PRs.